package.json

```diff
@@ -162,7 +162,7 @@
 		"@storybook/addons": "^6.5.9",
 		"@storybook/storybook-deployer": "^2.8.16",
 		"@storybook/vue": "^6.5.9",
-		"@testing-library/cypress": "^8.0.3",
+		"@testing-library/cypress": "^10.0.2",
```

Comment:
Are Cypress tests still passing?

Reply:
Tests are still running, but not all of the tests are passing in either this branch or `main`. Looks like we have at least some experiments leading to failures.